### ISSUE TITLE
Multiple BreakLine don't work on Text/RichText Widget

### ISSUE THREAD

**Issue by rodolfogoulart** [DavBfr/dart_pdf#1413]
**Multiple BreakLine don't work on Text/RichText Widget**

**To Reproduce**
Code snippet to reproduce the behavior:
```dart
pw.RichText(
            softWrap: true,
            text: pw.TextSpan(children: [pw.TextSpan(text: '\n'), pw.TextSpan(text: '\n Text with multiple breakline \n\n another text \n\n\n last text.')]),
            overflow: pw.TextOverflow.span,
          ),
```

**Expected behavior**
multiple breaklines on the text

**Desktop (please complete the following information):**
 - [ ] iOS
 - [ ] Android
 - [ ] Browser
 - [ x] Windows
 - [ ] Linux

**Comment by deepak-agendaboa** [comment]
See https://github.com/DavBfr/dart_pdf/issues/991

**Comment by DavBfr** [comment]
Fixed in pdf 3.10.7
